ckan/common.py

```diff
@@ -208,3 +208,41 @@ def _get_session():
 # Provide a `c`  alias for `g` for backwards compatibility
 g = c = LocalProxy(_get_c)
 session = LocalProxy(_get_session)
+
+# replacement for converters that previously came from paste.deploy.converters
+# [#4797]
+truthy = frozenset([u'true', u'yes', u'on', u'y', u't', u'1'])
+falsy = frozenset([u'false', u'no', u'off', u'n', u'f', u'0'])
+
+
+def asbool(obj):
+    if isinstance(obj, basestring):
```

Comment:
Can you change this to the following to ensure Python 3 compatibility?

```suggestion
    if isinstance(obj, six.string_types):
```

That should fix the Travis CI failure as well